packages/rpc-graphql/src/loaders/__tests__/account-loader-test.ts

```diff
@@ -66,4 +74,1264 @@ describe('account loader', () => {
             expect(rpc.getAccountInfo).toHaveBeenCalledTimes(2);
         });
     });
+    describe('batch loading', () => {
```

Comment:
I think you still need a test to exercise multiple batches being split off when the number of fetches exceeds the max batch size. 

Maybe create an RPC with the `maxMultipleAccountsBatchSize` hardcoded to 3 and then try to make 4 requests.

Reply:
I have one! It's generating a huge query with 150 keys. Now that I have the input argument at the `createRpcGraphQL` level, I'm going to rework this to use the input arg as you said.

https://github.com/solana-labs/solana-web3.js/blob/024daa22af032636dd7975c535cdaf36f1b9580b/packages/rpc-graphql/src/loaders/__tests__/account-loader-test.ts#L500

Reply:
Done!